lib/gui/models/store.js

```diff
@@ -189,6 +189,16 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         throw errors.createError(`Invalid state percentage: ${action.data.percentage}`);
       }
 
+      const HUNDRED = 100;
+      if (action.data.percentage > HUNDRED) {
+        throw errors.createError(`Excessive state percentage: ${action.data.percentage}`);
+      }
+
+      const ZERO = 0;
+      if (action.data.percentage < ZERO) {
+        throw errors.createError(`Deficient state percentage: ${action.data.percentage}`);
```

Comment:
I'd be tempted to make these all report the same `Invalid state percentage: ${action.data.percentage}` message?
Potentially you could even combine all three if-clauses into a single if-statement.